### PR TITLE
[orb] GitHub version

### PR DESCRIPTION
**Pull request by qlikmats**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,30 @@ jobs:
...
+    - image: circleci/golang:1.15
+    steps:
+      - checkout
+      - github/set-version
+
+workflows:
+  "Build my project":
+    jobs:
+      - generate-version
+```
```
Comment (ibiqlik) [inline README.md:68]: Should this be placed in `org-version/README.md` instead?

File: orb-version/orb.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+version: 2.1
+
+description: >
+  Sets github version info in a text file.
+display:
+  source_url: https://github.com/qlik-oss/ci-tools/orb/github-version
+
+commands:
+  set-version:
+    description: Generate SemVer format *.*.* or *.*.*-shortCommitID
```
Comment (ibiqlik) [inline orb-version/orb.yml:10]:
```suggestion
    description: Generate SemVer format *.*.* or *.*.*-x-g<shortCommitID>
```

The git describe format is *.*.*-<numberOfCommitsSinceTag>-g<shortCommitID>

File: orb-version/orb.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        default: 'version.txt'
+        description: >
+          Name of the file, where Github version info will be stored.
+        type: string
+    steps:
+      - run:
+          name: Generate version info
+          command: |
+            mkdir -p workspace/
+            version=$(git describe --tags --abbrev=7 --match "v*")
```
Comment (ibiqlik) [inline orb-version/orb.yml:27]:
```suggestion
            if [ -n "${CIRCLE_TAG}" ]; then
              version=${CIRCLE_TAG#v}
            else
              version=$(git describe --tags --abbrev=7 --match "v*")
              version=${version#v}
            fi
```
Something like this

**Review by ibiqlik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (qlikmats) [reply README.md:68]: yeah

File: orb-version/orb.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      file:
+        default: 'version.txt'
+        description: >
+          Name of the file, where Github version info will be stored.
+        type: string
+    steps:
+      - run:
+          name: Generate version info
+          command: |
+            mkdir -p workspace/
```
Comment (ibiqlik) [inline orb-version/orb.yml:26]:
```suggestion
            mkdir -p <<parameters.path>>
```
I guess?

**Review by ibiqlik — APPROVED** [review]

**Pull request closed without merging**